src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -340,44 +373,62 @@ protected override SupervisorStrategy SupervisorStrategy()
                 var directive = Directive.Stop;
 
                 ex.Match()
-                    .With<InvalidAddressAssociationException>(ia =>
```

Comment:
Renamed these exceptions so they match what appears on the JVM. These exceptions are marked as `internal` so it should cause no breaking changes.